open_flamingo/eval/evaluate.py

```
@@ -163,44 +212,51 @@ def main():
     module = importlib.import_module(f"open_flamingo.eval.models.{args.model}")
     eval_model = module.EvalModel(leftovers)
 
+    if len(args.trial_seeds) < args.num_trials:
```

should we raise an exception here instead?